source/features/reactions-avatars.tsx

```diff
@@ -54,7 +54,7 @@ async function showAvatarsOn(commentReactions: Element): Promise<void> {
 
 	const participantByReaction = select
 		.all(':scope > button', commentReactions)
-		.map(getParticipants);
+		.map(element => getParticipants(element));
```

Comment:
I kept the variable as element (1/2)

Reply:
These are buttons. We can also change the signature of the callback from "container: HTMLElement" to "button: HTMLButtonElement" I think

Reply:
Both done (regarding the latter, I think I did it ok).

Reply:
Sorry, I missed `HTMLButtonElement`. I'll revert and do a new commit.

Reply:
Done.